src/lib.rs

```diff
@@ -276,15 +294,8 @@ where
                 config.inbound_router_capacity,
                 config.inbound_router_max_idle_age,
             );
-            let tls_settings = config.tls_settings.as_ref().map(|settings| {
-                tls::ConnectionConfig {
-                    identity: settings.pod_identity.clone(),
-                    config: tls_server_config
-                }
-            });
             serve(
                 inbound_listener,
```

Comment:
Seeing a branch that _reduces_ the number of parameters passed to this function rather than increasing it is refreshing! :)